source/config/localhost_config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "address": "[::1]",
+    "port": 0,
```

Comment:
Consider keeping port 31763 so this is a localhost-only version of `insecure_config.json`.

Reply:
Added known port to localhost_config.json